### PR TITLE
Release 0.7.4

### PR DESCRIPTION
**Pull request by florimondmanca**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file.
...
+- Add Trio concurrency backend. (Pull #276)
+- Add `params` parameter to `Client` for setting default query parameters. (Pull #372)
+- Add support for `SSL_CERT_FILE` and `SSL_CERT_DIR` environment variables. (Pull #307)
+- Add debug logging to calls into ASGI apps. (Pull #371)
+- Add debug logging to SSL configuration. (Pull #378)
+
+### Fixed
+
+- Fix a bug when using `Client` without timeouts in Python 3.6. (Pull #383)
+- Propagate `Client` configuration to HTTP proxies. (Pull #377)
```
Comment (florimondmanca) [inline CHANGELOG.md]: @sethmlarson I'm not sure which category this should go into. Does this one look appropriate?

Comment (sethmlarson) [reply CHANGELOG.md]: Yep it looks good to me :) Fix is definitely the right one.

**Comment by florimondmanca** [comment]
Merging now, @sethmlarson feel free to `scripts/publish` since I won’t be able to do it before tomorrow. :)

**Comment by sethmlarson** [comment]
:+1: Will do, thanks @florimondmanca!

**Comment by sethmlarson** [comment]
Deploy complete! :tada:

**Pull request merged**